File: src/main/Java/boj_step9/boj11653.java
import java.io.*;

public class boj11653 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        long n = Long.parseLong(reader.readLine());
        int divisor = 2;
        while (n != 1) {
            if (n % divisor == 0) {
                writer.append(String.valueOf(divisor));
                writer.newLine();
                n /= divisor;
            } else divisor++;
        }
        writer.close();
    }
}

File: src/main/kotlin/boj_step9/boj11653.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import kotlin.math.sqrt

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    var n = r.readLine().toInt()
    for (i in 2 .. n) {
        while (n % i == 0) {
            w.write("$i\n")
            n /= i
        }
        if(n < 2) break
    }
    w.close()
}